lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -2132,9 +2140,17 @@ def _scrape_configs(self) -> dict:
                     "label_structured_data": True,
                     "labels": syslog_labels,
                 },
+                "relabel_configs": [
+                    {"source_labels": ["__syslog_message_{}".format(val)], "target_label": val}
+                    for val in relabel_mappings
+                ]
+                + [{"action": "labelmap", "regex": "__syslog_message_sd_(.+)"}],
             }
             scrape_configs.append(syslog_config)  # type: ignore
 
+        import pprint
```

Comment:
Is `pprint` such a large module that is preferable to import it here rather than at the beginning of the file?

Reply:
Gone now, but it's one of those "printf debugging" things in my workflow. If something is unclear, I import on the spot to log it so it's easier to remember to remove it later